### DISCUSSION TITLE
speed up first start on intranet server

### DISCUSSION THREAD

**Discussion by m-hew** [dbeaver/dbeaver#36011] [Q&A]
I use DBeaver 24.0.5 on an intranet server that is not connected to the internet. 
The first startup of the program takes 1-2 minutes where only the splash screen is shown. Once it started, I can close it and subsequent starts only takes seconds. Although this resets after a while. 

I can only assume that DBeaver tries to connect to some internet resource and then waits for a timeout before continuing. I tried to find all options that are linked to online connectivity, but didn't see any improvement.
Or maybe there is another cause. 

Is there any way how I can speed up the first startup?

**Comment by TB-HunBoka** [comment]
Could you please download the latest version 24.2.3, after starting, it will ask if you would like to add DBeaver as exception to the windows defender. Add it as an exception, and can you please check if this is still a problem?

**Reply by m-hew** [reply]
Sorry, I forgot that DBeaver is available for multiple systems. 
I'm running it on Rocky Linux 9. 
AV is Bitdefender. I can ask the IT if they make an exception to see if this changes anything.
